fortanix-sgx-tools/src/bin/ftxsgx-elf2sgxs.rs

```diff
@@ -201,6 +258,20 @@ impl<'a> LayoutInfo<'a> {
             bail!("ENCLAVE_SIZE symbol is not naturally aligned")
         }
 
+        if let (Some(EH_FRM_HDR_BASE), Some(EH_FRM_HDR_SIZE)) = (syms.EH_FRM_HDR_BASE, syms.EH_FRM_HDR_SIZE) {
+            check_size!(EH_FRM_HDR_BASE == 8);
+            check_size!(EH_FRM_HDR_SIZE == 8);
+        }
```

Comment:
Minor thing, but I believe it is standard practice to put the else on the same line as the closing bracket

Reply:
fixed, thanks